src/Client/app/(admin)/exams/create/page.tsx

```
@@ -154,132 +166,133 @@ export default function CreateExamPage() {
-                            min="0"
-                            value={formData.durationMinutes?.toString() ?? "0"}
+                            min="1"
+                            value={formData.durationMinutes.toString()}
                             onChange={(e) => {
-                                const value = Number(e.target.value);
-                                if (!isNaN(value) && value >= 1) {
-                                    setFormData({
-                                        ...formData,
-                                        durationMinutes: value,
-                                    });
+                                const value = parseInt(e.target.value);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)